tags/README.md

```diff
@@ -43,7 +43,9 @@ Ps: If you are facing some issue performing these steps, you could also check ou
 
 # App Description
 
-Select and display multiple arbitrary values as tags with this Tags field plugin. The tag is ideal when working with oraganized data structures such as categories and label, for example. However, it uses is unlimited.
+Select multiple arbitrary values with this tag-styled component.
```

Comment:
Maybe we can rephrase this? It's not really arbitrary; the user is restricted to a list of strings. Only the content in the string is arbitrary.